src/main/resources/templates/hint/detail.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html lang="cs" xmlns:th="http://www.thymeleaf.org">
 <head th:replace="fragments/head :: head('Detail nápovědy')">
+    <resources mapping="/resources/**" location="file:/upload/images" />
```

Comment:
Nejsem si jistý, ale nemělo by tohle jít taky do toho head fragmentu?

Reply:
Nakonec to není potřeba, tak jsem to dala pryč